modules/n1ql/pages/n1ql-language-reference/metafun.adoc

```diff
@@ -557,6 +631,88 @@ For examples showing how to index metadata information, refer to xref:n1ql-langu
 
 For examples showing how to use metadata information in the predicate of an ANSI JOIN clause, refer to xref:n1ql:n1ql-language-reference/join.adoc[JOIN Clause].
 
+[[node-name]]
+== NODE_NAME()
+
+=== Description
+
+Returns the name of the node on which the query is running.
+
+=== Arguments
+
+None.
+
+=== Return Value
+
+A string representing a node name.
+
+=== Example
+
+[[node-name-ex,NODE_NAME() Example]]
+====
+.Query
+[source,sqlpp]
+----
+SELECT NODE_NAME() AS node_name;
+----
+
+.Results
+[source,json]
+----
+[
+  {
+    "node_name": "127.0.0.1:8091"
+  }
+]
+----
+====
+
+[[node-uuid,NODE_UUID()]]
+== NODE_UUID(`expression`)
+
+=== Description
+
+Returns the UUID of a node.
+
+=== Arguments
+
+expression::
+A string, or an expression resolving to a string, representing a node name.
+To get the UUID of the node on which the query is running, use the empty string `""`.
+
+=== Return Value
+
+A string representing the node UUID.
+
+If the input expression is not a string, the return value is `null`.
+
+If the input expression is `missing`, the return value is also `missing`.
+
+=== Example
+
+[[node-uuid-ex,NODE_UUID() Example]]
+====
+.Query
+[source,sqlpp]
+----
+SELECT NODE_UUID("") AS from_empty_string,
+       NODE_UUID("127.0.0.1:8091") AS from_node_uuid,
```

Comment:
Should these not be "from_node_name"/"from_node_function" (or "uuid_from_node_name" etc.) ?

Reply:
Done